.env.dev

```diff
@@ -11,9 +11,9 @@ mailer_password=null
 secret=ThisTokenIsNotSoSecretChangeIt
 session_max_absolute_lifetime=3600
 session_max_relative_lifetime=600
-logout_redirect_url='https=//www.surf.nl/over-surf/werkmaatschappijen/surfnet'
+logout_redirect_url='https://www.surf.nl/over-surf/werkmaatschappijen/surfnet'
 # All users in these teams get the administrator role
-administrator_teams="'urn:collab:group:dev.openconext.local:dev:openconext:local:spd_admin'"
+administrator_teams="'urn:collab:group:dev.openconext.local:dev:openconext:local:spd_admin','urn:collab:org:dev.openconext.local'"
```

Comment:
I removed the 'default' team membership to prevent you are always admin when logging in. If this not breaks the tests, its OK to leave it like this! I can always change it if I want to. (just feedback to explain to you why the team is not there by default)

Reply:
@quartje this is about your commit